test/spec.yml

```diff
@@ -1215,3 +1215,35 @@
     -   in: bn create node8 alreadyExists
         exit: 1
         err: "/home/dockeruser/test/binaris.yml already exists"
+
+- test: create configuration options (good-path)
+  steps:
+    -  in: bn create node8 testConfOpts --config.executionModel=exclusive --config.env.PATH --config.env.FOO=bar
+    -  in: |-
+           cat <<EOF >>/home/dockeruser/test/expected.yml
+           functions:
+             testConfOpts:
+               file: function.js
+               entrypoint: handler
+               executionModel: exclusive
+               runtime: node8
+               env:
+                 PATH: null
```

Comment:
Also not sure I like the literal null here (in comparison to empty arg)

Reply:
null here is not a string but the value null in yaml. the generated yaml is legal and works as expected.
to actually have the value null you'd have to use `'null'`.
moreso, using our yaml package it's impossible to generate empty values.

Reply:
If `js-yaml` can't generate empty values then it's fine as is.